metricflow/query/query_parser.py

```diff
@@ -573,6 +583,8 @@ def _adjust_time_range_constraint(
                 self._time_granularity_solver.resolve_granularity_for_partial_time_dimension_specs(
                     metric_references=metric_references,
                     partial_time_dimension_specs=(partial_metric_time_spec,),
+                    read_nodes=self._read_nodes,
+                    node_output_resolver=self._node_output_resolver,
```

Comment:
Checkpoint, this must be changed in a later commit one way or the other.....

Reply:
What do you mean??